docs/stransmit.md

```diff
@@ -1,5 +1,5 @@
 SRT Live Transmit
----------
+-----------------
 
 The *srt-live-transmit* tool is a universal data transport tool, which's
 intention is to transport data between SRT and other medium.
```

Comment:
The *srt-live-transmit* tool is a universal data transport tool, the
intention of which is to transport data between SRT and other mediums.